qa/vagrant/src/test/resources/packaging/scripts/os_package.bash

```diff
@@ -81,7 +82,8 @@ verify_package_installation() {
     assert_file "$ESSCRIPTS" d root elasticsearch 750
     assert_file "$ESDATA" d elasticsearch elasticsearch 755
     assert_file "$ESLOG" d elasticsearch elasticsearch 755
-    assert_file "$ESPLUGINS" d elasticsearch elasticsearch 755
```

Comment:
It looks like I just screwed this up when I was implementing deb and rpm in gradle. Thanks for fixing it.